server/sonar-server/src/main/java/org/sonar/server/permission/ws/template/BulkApplyTemplateAction.java

```diff
@@ -91,6 +104,40 @@ public void define(WebService.NewController context) {
       .setDeprecatedKey(PARAM_QUALIFIER, "6.6");
 
     createTemplateParameters(action);
+
+    action
+      .createParam(PARAM_PROJECTS)
+      .setDescription("Comma-separated list of project keys")
+      .setSince("6.6")
+      .setExampleValue(String.join(",", KEY_PROJECT_EXAMPLE_001, KEY_PROJECT_EXAMPLE_002));
+
+    action
+      .createParam(PARAM_PROJECT_IDS)
```

Comment:
This parameter should not be added, it didn't exist before

Reply:
it existed in the `api/projects/search` WS

Reply:
Let's discuss this